src/model.jl

```diff
@@ -53,20 +63,21 @@ function PowerManagementProblem(fq::Vector, fl::Vector, d, pmax, gmax, A, B; τ=
         + (τ/2)*sumsquares(p)
     )
     add_constraints!(problem, [
-        -p <= pmax, #λpl
-        p <= pmax, #λpu
-        -g <= 0, #λgl
-        g <= gmax, #λgu
-        0 == A*p - B*g + d + ch - dis, #ν
+        -p <= pmax,
+        p <= pmax,
+        -g <= 0, 
+        g <= gmax,
+        0 == p - F*(B*g - d + ch - dis),
+        0 == ones(n)' * (B*g - d + ch - dis),
```

Comment:
You don't wanna add in comments the corresponding dual variables (e.g. `#\lambda pl`)? I thought it was rather clear and useful to know, later in the code, what variable referred to which constraint.

Reply:
I can add that.